concepts/05 UI Components/TagBox/00 Getting Started with TagBox/60 Configure Multiple Tags Selection.md

```diff
@@ -0,0 +1,71 @@
+Use the following properties to adapt the TagBox text field to multiple selected tags:        
+
+- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)     
+Specifies whether to display tags on multiple lines when the tags overflow the component's width.
+
+- [maxDisplayedTags](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#maxDisplayedTags)      
+Specifies the limit after which the TagBox replaces selected tags with a single multi-tag.
```

Comment:
Likewise, you need imperative mood, because it's a getting started topic, not some overview.